REFERENCE.md

```diff
@@ -0,0 +1,251 @@
+# Learnosity PHP-SDK: Reference guide and Examples
+
+## Usage
+
+There are three main classes:
+
+ * The Init class, which creates the signed security packets to initialise the JavaScript APIs;
+ * The Remote class, which encapsulates remote calls to the APIs and their responses;
+ * The DataAPI class, which allows your app to interact with the Data API from PHP code.
+
+### Examples
+
+A couple of basic examples are included in the `examples/` directory. You can run them with, e.g.,
+
+    php examples/simple/init_items.php
+
+or
+
+    php examples/simple/init_data.php
+
+You can find a complete PHP site with examples of Learnosity APIs integration in our [demos site](https://demos.learnosity.com/).
+
+You can download the entire site or browse the code directly on [github](https://github.com/Learnosity/learnosity-demos/).
+
+### Init
+
+The Init class is used to create the necessary *security* and *request* details used to integrate with a Learnosity API. Most often this will be a JavaScript object.
+
+``` php
+//Include the Init classes.
+use LearnositySdk\Request\Init;
+```
+
+The Init constructor takes up to 5 arguments:
+
+ * [string]  service type
+ * [array]   security details (**no secret**)
+ * [string]  secret
+ * [request] request details *(optional)*
+ * [string]  action *(optional)*
+
+``` php
+<?php
+
+// Instantiate the SDK Init class with your security and request data:
+$Init = new Init(
+   'questions',
+   [
+       'consumer_key' => 'yis0TYCu7U9V4o7M',
+       'domain'       => 'localhost',
+       'user_id'      => '$ANONYMIZED_USER_ID'
```

Comment:
Great opportunity to clean this up!

Let's uses `Uuid::generate()` to create a `$user_id`, and remove any use of `$ANONYMIZED_USER_ID` (which is particularly confusing in PHP, as if the string were double-quoted, it would get replaced by the value of the [empty] variable)

Reply:
Fixed. Although $ANONYMIZED_USER_ID still appears in some of the actual SDK software files, it's no longer in the docs or examples.